### DISCUSSION TITLE
Version 1.32.2

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#986] [Announcements]
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.32.2'>Version 1.32.2</a>.</em>

**Comment by sk425** [comment]
First off, thank you all for building this incredible app. Just a quick note: "Bug fixes" are ambiguous,  it leaves me guessing what's fixed. Without any knowledge of the code base, I wouldn't be able to tell what's happening looking at the diff's. If there could be some more reference to the issue(s) that were solved (or similar), that would be more helpful. Cheers.

**Reply by MM2-0** [reply]
True, I'm lazy. Normally I assume that the people who encountered a certain bug subscribed to the relevant issue, so they where already notified when the issue was closed. But that may not always be the case.

**Reply by muelsyse-az** [reply]
I wonder if there's a way to automate the release notes process so it lists all commits made for that release
